src/app_service/service/message/dto/message.nim

```diff
@@ -83,7 +83,6 @@ type TransactionParameters* = object
 type MessageDto* = object
   id*: string
   communityId*: string
-  whisperTimestamp*: int64
```

Comment:
You can leave it in the dto if you want

Reply:
Initially did it that way, with the comment `# don't use`, then decided to remove it altogether. Will bring it back if ever needed.